### PR TITLE
Fix PrettyCI

### PR DESCRIPTION
**Pull request by RocKordier**

### CODE REVIEW

File: Classes/Twig/Extension/DBRelationExtension.php [hunk 1]
```diff
@@ -70,8 +70,12 @@ public function lookupRelation(EnvironmentTwig $env, \Tx_Rnbase_Domain_Model_Bas
...
         $field = $env->getConfigurations()->get($confId . 'join.field');
         if (! $alias || ! $field) {
-            throw new \Exception('Verify configuration for relation \'' . htmlspecialchars($arguments['relation']) . '\'.
-                            Table alias or field not found. Full typoscript path: ' . $confId);
+            throw new \Exception(sprintf(
+                "Verify config for relation '%s' Table alias or field not found. Full typoscript path: %s",
+                    htmlspecialchars($arguments['relation']),
+                    $confId
+                )
+            );
```
Comment (rengaw83) [inline Classes/Twig/Extension/DBRelationExtension.php]: There are still errors in lines [73-77](https://rc.dmknet.de/group/Team-4?msg=xTzWvviauk66rdv7j).

File: .php_cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+$finder = \PhpCsFixer\Finder::create()
+    ->in(__DIR__.'/Classes');
+return PhpCsFixer\Config::create()
+    ->setFinder($finder)
+    ->setRiskyAllowed(true)
```
Comment (Baachi) [inline .php_cs]: Should be avoided, i think.

File: Classes/ContentObject/TwigContentObject.php [hunk 3]
```diff
@@ -76,19 +80,19 @@ public function render(
...
+        $contextData = $this->getContext($configurations);
 
         $content .= $renderer->render($contextData);
 
         return $content;
     }
 
     /**
-     * Builds the  configuration object based on the conf
+     * Builds the  configuration object based on the conf.
```
Comment (Baachi) [inline Classes/ContentObject/TwigContentObject.php]: Can you remove the double space?

**Review by rengaw83 — COMMENTED** [review]
There are still errors in Twig/Extension/DBRelationExtension.php Line 73-77 and Classes/Cache/TYPO3Cache.php Line 55
Can yout fix this please and add `Classes/ViewHelpers/BaseViewHelper.php` ti ignore in `.php_cs`

File: Classes/Twig/Extension/DBRelationExtension.php (hunk 1 above)
Comment (rengaw83) [reply Classes/Twig/Extension/DBRelationExtension.php]: Keep calm and force push :P

File: Classes/Twig/Extension/DBRelationExtension.php [hunk 4]
```diff
@@ -28,57 +29,60 @@
...
-     * @param array $arguments
+     * @param string                           $paramName
+     * @param array                            $arguments
      *
      * @return mixed|null
      */
     public function lookupRelation(EnvironmentTwig $env, \Tx_Rnbase_Domain_Model_Base $entity, array $arguments = [])
     {
-        $confId = $env->getConfId() . 'relations.' . htmlspecialchars($arguments['relation']) . '.';
+        $confId = $env->getConfId().'relations.'.htmlspecialchars($arguments['relation']).'.';
```
Comment (Baachi) [inline Classes/Twig/Extension/DBRelationExtension.php]: Wouldn't it be better to use `sprintf` here?

**Review by Baachi — CHANGES_REQUESTED** [review]

**Review by rengaw83 — APPROVED** [review]

**Review by Baachi — APPROVED** [review]

**Pull request merged**